functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/affected/LocalChangesAffectedTestsSelectionExecutionFunctionalTest.java

```diff
@@ -30,16 +35,17 @@ public void should_only_execute_tests_related_to_single_local_change_in_producti
     }
 
     @Test
-    public void should_only_execute_tests_related_to_multiple_local_changes_in_production_code_when_affected_is_enabled() throws Exception {
+    public void should_only_execute_tests_related_to_multiple_local_changes_in_production_code_when_affected_is_enabled()
+        throws Exception {
         // given
         project.configureSmartTesting()
-                    .executionOrder(AFFECTED)
-                    .inMode(SELECTING)
+            .executionOrder(AFFECTED)
+            .inMode(SELECTING)
```

Comment:
I guess that it is a result of auto-formatting process. To be honest, I would do the same thing as Dipak did. I would rather keep formatting specified in `.editorconfig` file